deepchem/feat/material_featurizers/lcnn_featurizer.py

```diff
@@ -0,0 +1,656 @@
+import numpy as np
+from deepchem.feat import Featurizer
+from collections import defaultdict
+
+
+class LCNNFeaturizer(Featurizer):
+  """
+  Calculates the 2-D Surface graph features in 6 diffrent permutaions-
+
+  Based on the implementation of Lattice Graph Convolution Nueral
+  Network (LCNN) . This method produces the Atom wise features ( One Hot Enconding)
+  and Ajacent neighbor in the specificed order of permutations. Neighbors are determined
+  distance metric and Each Permutation of the Neighbors are calculated
+  is manner in which , First element is the node itself followed by a randomly
+  selected neighboring site. The next site shares a surface Pt atom with the previous
+  site. And they are picked consecutively.
+  First the template of the Primitive cell needs to be defined and then Each
+  structure(Data Point or diffrent condifuration of adsorbate atoms) is passed
+  for featurization.
+
+  The permuted neighbors are calculated using the Primitive cells i.e periodic cells
+  in all the data points are built via lattice transformation of the primitive cell.
+
+  [1] The Primitive Template file must be passed as raw text string or path file
+  [2] The datapoint must be passed as raw text string.
+
+  References
+  ----------
+  [1] Jonathan Lym and Geun Ho Gu, J. Phys. Chem. C 2019, 123, 18951−18959
+
+  Examples
+  ----------
+  The input format for primitive cell Template is stored in a file named
```

Comment:
Could you check that this section renders correctly by building the docs locally on your machine? I'm not sure this is formatted correctly.